static/app/views/automations/components/automationListRow.tsx

```diff
@@ -51,6 +52,9 @@ export function AutomationListRow({
       <CellWrapper className="action">
         <ActionCell actions={actions} disabled={disabled} />
       </CellWrapper>
+      <CellWrapper className="projects">
+        <ProjectList projectSlugs={[]} />
+      </CellWrapper>
```

Comment:
Should this be using the `useAutomationProjectIds` hook? We could also do this in a separate PR since we also have to update the column dropping for different screen sizes

Reply:
Good catch! Fixed in 0d1ad7a1f11